x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/SqlFunctionTypeRegistry.java

```diff
@@ -18,7 +18,8 @@
     private enum Types {
         CONDITIONAL(ConditionalFunction.class),
         GROUPING(GroupingFunction.class),
-        SCORE(Score.class);
+        SCORE(Score.class),
+        ARRAY(Array.class);
```

Comment:
I think this list is meant to be ordered alphabetically.

Reply:
Not really, the functions within the groups are of course ordered, but the order of the groups it's up to us I think.

Reply:
I've reordered them, just in case.